common/core/web/keyboard-processor/tests/cases/basic-init.js

```diff
@@ -2,12 +2,15 @@ var assert = require('chai').assert;
 var fs = require("fs");
 var vm = require("vm");
 
-let KeyboardProcessor = com.keyman.text.KeyboardProcessor;
-global.keyman = {};
-global.com = com;
+let KeyboardProcessor = require('../../build/index.bundled.js');
+
+// Required initialization setup.
+global.com = KeyboardProcessor.com; // exports all keyboard-processor namespacing.
+global.keyman = {}; // So that keyboard-based checks against the global `keyman` succeed.
+                    // 10.0+ dependent keyboards, like khmer_angkor, will otherwise fail to load.
 
 // Initialize supplementary plane string extensions
-String.kmwEnableSupplementaryPlane(false);
+String.kmwEnableSupplementaryPlane(false);    
```

Comment:
Can you turn on an option in your editor to trim whitespace on end of lines? That avoids this kind of whitespace back-and-forth :grin: